src/scitokens/scitokens.py

```diff
@@ -211,6 +211,23 @@ def __contains__(self, claim):
             return True
         return False
 
+    def __delitem__(self, claim):
+        """
+        Delete the claim from the SciToken
+        """
+        deleted = False
+        if claim in self._claims:
+            del self._claims[claim]
+            deleted = True
+        if claim in self._verified_claims:
+            del self._verified_claims[claim]
+            deleted = True
+        
+        if deleted:
+            return deleted
```

Comment:
```suggestion
            return
```
You can't check the return value of a deletion (`x = del foo["a"]` is a SyntaxError) so there's not much point in returning anything.